### ISSUE TITLE
Accessibility bucket

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4525]
Total 10: 3 need fixes, 2 investigates, 3 need re-verify, 2 won't fix.

- Fix: https://msazure.visualstudio.com/CCI/_workitems/edit/15602765
   - https://github.com/microsoft/BotFramework-WebChat/issues/4557
   - Instead of narrating left/right, it should narrate previous/next
- Fix: https://msazure.visualstudio.com/CCI/_workitems/edit/15603122
   - https://github.com/microsoft/BotFramework-WebChat/issues/4558
   - In High Contrast mode, the "Retry" button/link should be in hyperlink color
- Re-verify: https://msazure.visualstudio.com/CCI/_workitems/edit/15627408
   - "Retry" button is not narrated through live region
   - We should have fixed it in #4362 of 4.15.3 (August)
   - Update: closed
- Fix: https://msazure.visualstudio.com/CCI/_workitems/edit/15627465
   - https://github.com/microsoft/BotFramework-WebChat/issues/4559
   - The color use in SVG should be a CSS system color of "text color"
- Won't fix: https://msazure.visualstudio.com/CCI/_workitems/edit/15627489
   - We cannot move focus because it will interrupt live region
   - Update: closed
- Investigate: https://msazure.visualstudio.com/CCI/_workitems/edit/15627650
   - https://github.com/microsoft/BotFramework-WebChat/issues/4560
   - When in browse mode and screen reader is reading the whole page, for messages that has "Send failed. Retry", screen reader actually still read it as "Sending"
- Re-verify: https://msazure.visualstudio.com/CCI/_workitems/edit/15627745
   - This should be fixed in 4.15.3
   - We turned the suggested actions container into a button toolbar
   - Update: closed
- Won't fix: https://msazure.visualstudio.com/CCI/_workitems/edit/15627752
   - This is expected
   - Adaptive Cards content can be very complex and narrating very complex content as a label (a.k.a. a single text string) is causing a lot of issues in many screen readers we tested
   - Adaptive Cards also did not offer an alt text
   - So we end up just telling the user there is an attachment, then need to press <kbd>ENTER</kbd> to read it
   - Update: closed
- Re-verify: https://msazure.visualstudio.com/CCI/_workitems/edit/15627779
   - This should be fixed in 4.15.3
   - Suggested actions should be narrated thru live region
   - Update: closed
- Investigates: https://msazure.visualstudio.com/CCI/_workitems/edit/15627865
   - https://github.com/microsoft/BotFramework-WebChat/issues/4561
   - This is partially fixed in 4.15.5, however, it still have an issue
      1. Turn on Narrator
      1. Press <kbd>TAB</kbd> to focus on the keyboard help screen
      1. Press down arrow key
      1. Expect: It should read the title of the dialog "Keyboard controls"
      1. Actual: It read the subtitle of the dialog "Chat window", skipped the title

**Comment by compulim** [comment]
Closing this bucket bug as we expanded them.
